README.adoc

```diff
@@ -129,20 +129,20 @@ However, you can externalize configuration data out of the application package,
 
 Custom configuration sources can be created by implementing the `org.eclipse.microprofile.config.spi.ConfigSource` interface and using the `java.util.ServiceLoader` mechanism.
 
-A `CustomConfigSource.json` JSON file has already been created outside of the application (`guide-microprofile-config/resources/`).
+A `CustomConfigSource.json` JSON file has already been created outside of the `start` directory (`guide-microprofile-config/resources/CustomConfigSource.json`).
 This file contains 4 custom config properties and has an ordinal of `50`.
 To use these properties in the application, the data object needs to be transformed from this JSON file to the configuration for your application.
 
-To link this JSON file to your application, create a `start/src/main/java/io/openliberty/guides/config/CustomConfigSource.java` file and add the following content to override the `ConfigSource` interface:
+To link this JSON file to your application, create a `src/main/java/io/openliberty/guides/config/CustomConfigSource.java` file in the `start` directory. Add the following content to implement the `ConfigSource` interface:
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/config/CustomConfigSource.java[tags=customConfig]
 ----
 
-The `getProperties()` method reads the key value pairs from the `resources/CustomConfigSource.json` JSON file and writes the information into a map.
+The `getProperties()` method reads the key value pairs from the `guide-microprofile-config/resources/CustomConfigSource.json` JSON file and writes the information into a map.
```

Comment:
I don't see we use the repo name in the file path anywhere else or in any other guides, "guide-microprofile-config/".. I can see this `resources/CustomConfigSource.json` is outside of the `start` folder. There are lots of jumping back of forth from the `start` folder for this guide.